cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/SymbolResolver.kt

```diff
@@ -398,6 +400,17 @@ open class SymbolResolver(ctx: TranslationContext) : ComponentPass(ctx) {
         val callee = call.callee
         val language = call.language
 
+        // Check for a possible overloaded operator->
+        if (isOverloadedOperator(callee, "->")) {
```

Comment:
I actually meant extracting the whole resolveOpreator, best viable, etc. stuff. isn't this 1:1 the same in both cases?

Reply:
Yes, basically, I was just thinking about the type update in the `resolveMember` case.
I now solved it as optional return type.
I don't see where the type should be used in the `handleCallExpression` case. Do you?

Reply:
Looks good, just add some documentation now and we're done :)

Reply:
Added doc for function `resolveOverloadedArrowOperator`